group-coordinator/src/main/java/org/apache/kafka/coordinator/group/consumer/ConsumerGroup.java

```diff
@@ -141,7 +141,7 @@ public static class TimeAndEpoch {
      * after having refreshed the metadata but the write operation failed. In this case, the
      * time is not automatically rolled back.
      */
-    private TimeAndEpoch nextMetadataRefreshTime = TimeAndEpoch.EMPTY;
```

Comment:
I renamed this one. It seems that `deadline` is more appropriate than `nextTime`.

Reply:
i noticed this across some places.

can we rework the comments in `hasMetadataExpired()` and in the test `testNextMetadataRefreshTime()` along with the name to use deadline?